userguide/system.rst

```diff
@@ -779,8 +780,9 @@ are unlocked with their password and any remaining devices, without a
 device-specific password, are unlocked using the global password.
 
 To configure a global password, go to
-:menuselection:`System --> Advanced --> SED Password` and input the
-password. Be sure to record the password and store it in a safe place!
+:menuselection:`System --> Advanced --> SED Password` and enter the
+password. Recording the password and storing it in a safe place is
+recommended.
 
```

Comment:
remove "and enter the password." it's redundant.
s/Recording the password and storing it in a safe place is recommended./Storing a copy of the password in a safe place is recommended./